### PR TITLE
Updated README.md to reference re-release and planning phase as per #810

### PR DESCRIPTION
**Pull request by evancofer**
This a commit to announcing the second project iteration. This should *almost certainly undergo additional discussion and/or editing from others* (esp @agitter @cgreene @stephenra ) before merging.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,10 +14,13 @@ The project operates on an open contribution model, welcoming contributions from
 To see what's incoming, check the open [pull requests](https://github.com/greenelab/deep-review/pulls).
 For project discussion and planning see the [Issues](https://github.com/greenelab/deep-review/issues).
 
-### Current stage: resubmitted to _JRSI_ and _bioRxiv_
+### Current stage: planning Deep Review 2
```
Comment (cgreene) [inline README.md]: Do you want to tag it with a year like _2019_ to make it clearer when you plan for the next release? I am also fine with 2, so this is just a suggestion.

File: README.md [hunk 2]
```diff
@@ -14,10 +14,13 @@ The project operates on an open contribution model, welcoming contributions from
...
 
 We [completed](https://github.com/greenelab/deep-review/releases/tag/v0.10) manuscript revisions in [response](content/response-to-reviewers.md) to two external peer reviews.
-Please see [issue #810](https://github.com/greenelab/deep-review/issues/810) for a discussion about future plans and whether we will continue to accept new pull requests.
+As of writing, we are aiming to publish an update of the deep review each year.
+The first release will be sometime in 2019.
```
Comment (cgreene) [inline README.md]: Next release? And maybe plan for something like June for a release + then pursue publication?

File: README.md [hunk 3]
```diff
@@ -14,10 +14,13 @@ The project operates on an open contribution model, welcoming contributions from
...
+Like the initial release, this should be an open and collaborative effort.
```
Comment (cgreene) [inline README.md]: Maybe:
> Like the initial release, we plan for an open and collaborative process.

File: README.md [hunk 4]
```diff
@@ -37,7 +40,7 @@ git pull --rebase upstream master
 
 ### Headline review format
 
-The manuscript is intended to be a headline review for [_Journal of the Royal Society Interface_](http://rsif.royalsocietypublishing.org/) on a topic overlapping the computer and life sciences in the area of systems pharmacology.
+The initial manuscript is intended to be a headline review for [_Journal of the Royal Society Interface_](http://rsif.royalsocietypublishing.org/) on a topic overlapping the computer and life sciences in the area of systems pharmacology.
```
Comment (cgreene) [inline README.md]: Maybe convert to:

> The initial manuscript was a headline review for...

**Comment by agitter** [comment]
@evancofer I should have time to review later today or tomorrow

**Comment by evancofer** [comment]
@cgreene I went through and clarified some things as per your suggestions. Thanks again for the feedback.

File: README.md [hunk 5]
```diff
@@ -14,10 +14,13 @@ The project operates on an open contribution model, welcoming contributions from
 To see what's incoming, check the open [pull requests](https://github.com/greenelab/deep-review/pulls).
 For project discussion and planning see the [Issues](https://github.com/greenelab/deep-review/issues).
 
-### Current stage: resubmitted to _JRSI_ and _bioRxiv_
+### Current stage: planning Deep Review 2019
 
 We [completed](https://github.com/greenelab/deep-review/releases/tag/v0.10) manuscript revisions in [response](content/response-to-reviewers.md) to two external peer reviews.
-Please see [issue #810](https://github.com/greenelab/deep-review/issues/810) for a discussion about future plans and whether we will continue to accept new pull requests.
+As of writing, we are aiming to publish an update of the deep review each year, with the next such release occuring in June 2019.
+We will continue to make project preprints available on bioRxiv, and aim to continue publishing the finished reviews in a peer-reviewed venue as well.
```
Comment (agitter) [inline README.md]: As a practical matter, we won't be able to update the manuscript in bioRxiv.  We would have to create a new version as a new entity or use a different preprint server.  Either could be good options.  What you have written is accurate though, I don't suggest changing it.

It is also unknown at this point which publishers would be interested in an updated version.  I suspect it depends heavily on how much the manuscript is updated.

File: README.md [hunk 6]
```diff
@@ -14,10 +14,13 @@ The project operates on an open contribution model, welcoming contributions from
 To see what's incoming, check the open [pull requests](https://github.com/greenelab/deep-review/pulls).
 For project discussion and planning see the [Issues](https://github.com/greenelab/deep-review/issues).
 
-### Current stage: resubmitted to _JRSI_ and _bioRxiv_
+### Current stage: planning Deep Review 2019
 
 We [completed](https://github.com/greenelab/deep-review/releases/tag/v0.10) manuscript revisions in [response](content/response-to-reviewers.md) to two external peer reviews.
-Please see [issue #810](https://github.com/greenelab/deep-review/issues/810) for a discussion about future plans and whether we will continue to accept new pull requests.
```
Comment (agitter) [inline README.md]: You could remove line 19 about `We completed manuscript revisions` if this section is about the next release.

Comment (evancofer) [reply README.md]: This is a good idea. Thanks.

**Comment by evancofer** [comment]
@stephenra Do you approve these changes?

**Comment by stephenra** [comment]
@evancofer LGTM

**Pull request merged**